### PR TITLE
Allow paddling fitBounds options to zoomToBounds

### LINKED ISSUE #775: Allow fitBounds options to be passed.
I'd like to pass options to the `fitBounds` method triggered by `zoomToBoundsOnClick`: http://leafletjs.com/reference.html#map-fitboundsoptions

...specifically looking to pass the `padding` option.

### PR DESCRIPTION
**Pull request by timkelty**
Fixes #775

### CODE REVIEW

File: src/MarkerCluster.js [hunk 1]
```diff
@@ -44,7 +44,9 @@ L.MarkerCluster = L.Marker.extend({
 	},
 
 	//Zoom to the minimum of showing all of the child markers, or the extents of this cluster
-	zoomToBounds: function () {
+	zoomToBounds: function (fitBoundsOptions) {
+        fitBoundsOptions = fitBoundsOptions || {};
```
Comment (danzel) [inline src/MarkerCluster.js]: I don't think this is necessary? We'll just pass undefined through which is totally fine.

**Comment by timkelty** [comment]
@danzel Ok, changes made.

**Comment by danzel** [comment]
Thanks!

**Comment by timkelty** [comment]
Thank you - could you publish a new version to NPM?

**Comment by danzel** [comment]
Done :)

**Pull request merged**